shell.nix

```diff
@@ -0,0 +1,27 @@
+let
+  localLib = import ./lib.nix;
+in
+{ system ? builtins.currentSystem
+, config ? {}
+, pkgs ? (import (localLib.fetchNixPkgs) { inherit system config; })
+}:
+
+let 
+  plutusPkgs = import ./. {};
+  ghc = pkgs.haskell.packages.ghc822.ghcWithPackages (ps: [
```

Comment:
this is using the unpatched ghc, rather then the patched ghc

Reply:
yeah... do we need the patched here? We're not building binaries for OSX currently either, we're just running tests on linux. Maybe I just pull that attribute out completely and we revisit later if we do need to patch GHC for OSX building. That's what I did in my most recent commit.